index.bs

```diff
@@ -589,6 +593,10 @@ Meeting Scheduling and Announcements</h5>
 	about the date and location of a meeting.
 	Shorter notice for a meeting is allowed
 	provided that there are no objections from group participants.
+	In the case of meetings
+	whose participation is not tied to any pre-existing [=Group=],
+	such as [=Workshops=],
+	shorter notice is not allowed.
```

Comment:
Do we want to forbid this completely, or allow for exceptions via e.g. approval of the CEO? I think we might want to allow exceptions...

Reply:
As discussed on the call, I've modified the PR to keep only applying this stricter requirement to workshops themselves, and not to a generalized category of open meetings.